osu.Game/Overlays/Dialog/PopupDialogDangerousButton.cs

```diff
@@ -47,16 +51,75 @@ public DangerousConfirmContainer()
             {
             }
 
+            private Sample tickSample;
+            private Sample confirmSample;
+            private bool isTicking;
+            private double lastTickPlaybackTime;
+            private AudioFilter lowPassFilter = null!;
+
+            [BackgroundDependencyLoader]
+            private void load(AudioManager audio)
+            {
+                lowPassFilter = new AudioFilter(audio.SampleMixer);
```

Comment:
Can you actually notice this one? In some initial testing, the `DialogOverlay` itself already adds a low pass filter when a dialog is displayed, and I'm not sure this one is doing much as a result (unless I'm missing some other case?)

Reply:
Aha, fair enough.

Reply:
I've rewritten your implementation of sample handling in this method, and fixed the `AudioFilter` not being added to the hierarchy (you were lucky that it was working because you didn't specify a duration for the transforms).

Please check it still seems okay.